pick.lic

```diff
@@ -112,8 +112,9 @@ class LockPicker
   end
 
   def buy_lockpick(type)
-    fput("buy #{type} lockpick")
-    fput("offer #{@lockpick_costs[type]}")
```

Comment:
Its also used to error check if the specified lockpick type is valid on lines 88-91